tools/attrd_updater.c

```diff
@@ -205,6 +214,13 @@ static GOptionEntry addl_entries[] = {
       INDENT "value, or the value set by a later request).  Default is 'no'.",
       "UNTIL" },
 
+    { "utilization", 'z', G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, attr_set_type_cb,
+      "When creating a new attribute, create it as a node utilization attribute\n"
+      INDENT "instead of an instance attribute.  If the attribute already exists,\n"
+      INDENT "its existing type (utilization vs. instance) will be used regardless.\n"
+      INDENT "(with -B, -U, -Y)",
+      NULL },
+
```

Comment:
Will crm_attribute --utilization --lifetime=reboot automatically work now?

Reply:
It doesn't look like --lifetime= does anything.  nodes/forever sets the pcmk__node_attr_perm flag, and status/reboot clears it.  But it's never actually used anywhere.

Reply:
Not attrd_updater, crm_attribute

Reply:
```
[root@rhel8-cluster-1 pacemaker]# crm_attribute --utilization --lifetime=reboot --name BBB --update 222
[root@rhel8-cluster-1 pacemaker]# cibadmin -Q | less
...
      <transient_attributes id="1">
        <instance_attributes id="status-1">
          <nvpair id="status-1-BBB" name="BBB" value="222"/>
        </instance_attributes>
        <utilization id="status-1">
          <nvpair id="status-1-AAA" name="AAA" value="123"/>
        </utilization>
      </transient_attributes>
```

So close.

Reply:
Looks like something's still wrong in crm_attribute:
```
(pcmk__send_ipc_request)        trace: ipc-sent   <create_attrd_op t="attrd" src="crm_attribute" attr_name="BBB" attr_set_type="instance_attributes" task="update" attr_value="222" attr_host="rhel8-cluster-1" attr_is_remote="0" attr_is_private="0"/>
```

vs when run from attrd_updater:
```
(pcmk__send_ipc_request)        trace: ipc-sent   <create_attrd_op t="attrd" src="attrd_updater" attr_name="AAA" attr_set_type="utilization" task="update" attr_value="123" attr_is_remote="0" attr_is_private="0"/>
```

Note the attr_set_type attribute.  I can fix that as part of this PR, or we can do a second one.  Whichever.

Reply:
Let's do it with this for completeness

Reply:
No problem.